core/src/main/java/org/sonarsource/sonarlint/core/plugin/PluginCopier.java

```diff
@@ -60,11 +63,18 @@ public PluginCopier(PluginCache fileCache, PluginIndexProvider pluginIndexProvid
     for (PluginReference pluginReference : pluginReferences) {
       File jarFile = getFromCacheOrCopy(pluginReference);
       PluginInfo info = PluginInfo.create(jarFile);
-      infosByKey.put(info.getKey(), info);
+      if ((info.isSonarLintSupported() != null && info.isSonarLintSupported().booleanValue()) || isWhitelisted(info.getKey())) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'isSonarLintSupported' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)